omnibus/files/private-chef-cookbooks/private-chef/templates/default/pg_hba.conf.erb

```diff
@@ -64,6 +64,8 @@
 # "local" is for Unix domain socket connections only
 local   all         <%= node['private_chef']['postgresql']['username'] %>                               peer
 
+host    all         <%= node['private_chef']['postgresql']['db_superuser'] %>         127.0.0.1/32           md5
```

Comment:
Are we able to specify localhost generically vs 127.0.0.1/32 here?  In particular, I'd like to avoid any IPv4 vs IPv6 problems down the road.